locale/fi/alert.json

```diff
@@ -0,0 +1,53 @@
+{
+  "title": "Heräte (Alert)",
+  "intro": "Heräte-komponentin avulla voidaan antaa käyttäjälle tietoa palvelun tilasta. Tällaista tietoa voi olla esimerkiksi häiriötilanteet tai ongelmat sivuston tai sen osan toiminnassa sekä tiedot tulevista käyttökatkoista.",
+  "alertHeading": "Alert version koodi-esimerkit",
+  "inlineAlertHeading": "InlineAlert version koodi-esimerkit",
+  "note.title": "Saavutettavuus ja käytettävyys",
+  "note.items": [
+    "Käytä komponentista aina asiaankuuluvaa varianttia (Alert tai InlineAlert). Ks. 'Koko ja käyttö'.",
+    "Alert komponentti on myös semanttisesti alert, joka tarkoittaa, että sen sisällön muuttuessa komponentin sisältö luetaan välittömästi ruudunlukijakäyttäjille.",
+    "InlineAlert komponentissa ruudunlukijakäyttäytyminen on säädettävissä, mutta sitä ei tule kytkeä kokonaan pois päältä."
+  ],
+  "contentParagrah": "Palvelussa käyttökatko ensi tiistaina 3:00 - 5:00 aamuyöllä",
+  "closeButton": "Sulje",
```

Comment:
These might be clearer if labeled something like `alertExample.contentParagraph` etc.

Reply:
fixed

Reply:
changed to "example.infoAlertParagraph"

Reply:
now there's `infoAlertParagraph` and `infoParagraph`. The distinction what content each of them refers to is still not clear.

Reply:
"example.inlineAlertInfoParagraph" now